devicetypes/smartthings/zwave-battery-thermostat.src/zwave-battery-thermostat.groovy

```diff
@@ -0,0 +1,669 @@
+/**
+ *  Copyright 2018 SmartThings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ */
+metadata {
+	definition (name: "Z-Wave Battery Thermostat", namespace: "smartthings", author: "SmartThings") {
+		capability "Actuator"
+		capability "Temperature Measurement"
+		capability "Thermostat"
+		capability "Refresh"
+		capability "Sensor"
+		capability "Health Check"
+		capability "Battery"
+		
+		attribute "thermostatFanState", "string"
+
+		command "switchMode"
+		command "switchFanMode"
+		command "lowerHeatingSetpoint"
+		command "raiseHeatingSetpoint"
+		command "lowerCoolSetpoint"
+		command "raiseCoolSetpoint"
+
+		fingerprint deviceId: "0x08"
```

Comment:
Should take this one out as it will put it in contention with zwave-thermostat as the fallback generic thermostat handler.